probe-rs/src/config/memory.rs

```diff
@@ -0,0 +1,117 @@
+/// Represents a region in flash.
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct FlashRegion {
+    pub range: core::ops::Range<u32>,
+    pub is_boot_memory: bool,
+    pub sector_size: u32,
+    pub page_size: u32,
+    pub erased_byte_value: u8,
+}
+
+impl FlashRegion {
+    /// Returns the necessary information about the sector which address resides in
+    /// if the address is inside the flash region.
+    pub fn get_sector_info(&self, address: u32) -> Option<SectorInfo> {
```

Comment:
It's idiomatic to leave out the `get_` prefix of getter-like methods.

Reply:
I am still not used to this ... always doing it wrong ... Also, that HashMap, Vec, etc. are still using get() (for obv. reasons) this is not easier to relearn :P